assets/sass/local.scss

```diff
@@ -291,7 +291,14 @@
   white-space: nowrap;
 }
 
-.govuk-input__wrapper {
-  width: 215px;
-  margin-bottom: 24px;
```

Comment:
Unrelated - this snuck in to customise layout of date picker on Visits by date page. But, it's too broad and affected other `input` elements. Resolved this with changes above to `bapv-another-date` classes.